.github/workflows/tests-r.yml

```diff
@@ -0,0 +1,47 @@
+name: R Tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      max-parallel: 4
+      fail-fast: false
+      matrix:
+        python-version: ['3.9']
+        platform: [ubuntu-latest]
+
+    runs-on: ${{ matrix.platform }}
+
+    if: contains(github.event.pull_request.labels.*.name, 'pr:test-r')
```

Comment:
This makes them opt-in right?

Reply:
Yes, needs to be labeled as `pr:test-r` (we have a similar `pr:test-nursery` label for the nursery tests). I checked that without adding the label the workflow indeed skips